cmd/healthcheck/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2019, 2022 The Swedish Internet Foundation
+// Copyright © 2019, 2022, 2023 The Swedish Internet Foundation
```

Comment:
Do we really need years here? Perhaps in the LICENSE file but IMHO it will get unwieldy quick (and I do not think it is needed for us to actually claim copyright).